2022/incoming/the-christmas-time-machine.pod

```diff
@@ -0,0 +1,81 @@
+Author: Thibault DUPONCHELLE <thibault.duponchelle@gmail.com>
+Title: The Christmas time machine
+Topic: Feature::Compat::Try
+
+=encoding utf8
+
+=head1 The Christmas time machine
+
+=head2 Terrible flashback at the line work
```

Comment:
Can you please explain what the word flashback means? I understand this like in a movie, when you see a brief bit of what happened in the past. Is that what you mean?

Reply:
Exactly

Reply:
"flashback" is fine for me, instead it is "at the line work" which reads oddly. Maybe it is an Americanism?